packages/aws-cdk/lib/init-templates/sample-app/java/src/main/java/com/myorg/%name.PascalCased%App.template.java

```diff
@@ -0,0 +1,13 @@
+package com.myorg;
+
+import software.amazon.awscdk.core.App;
+
+public final class %name.PascalCased%App {
+    public static void main(final String argv[]) {
```

Comment:
args?

Reply:
Actually, this brackets style (`String argv[]`) causes a warning in Intellij: "C-style array declaration". The more idiomatic Java way is to say `String[] args`.

Reply:
@skinny85 yes that's what I meant by that comment. might just pull this down and make that change myself